docs/.vuepress/config.js

```diff
@@ -38,6 +38,10 @@ module.exports = {
       index: "cosmos-sdk"
     },
     versions: [
+      {
+        "label": "v0.40",
+        "key": "v0.40"
```

Comment:
Now that the v0.40 has been kind of yanked, should we just show v0.39, v0.41 and master?

Reply:
if we don't plan on using v0.40, then we shouldn't include it in the version dropdown. Can you confirm that? @amaurymartiny

Reply:
confirmed

Reply:
Question about this. A recent bugfix (#8461) makes 0.40.* and 0.41.* buggy for ALL chains except gaia. Maybe in the docs we should just skip 0.40 and 0.41 altogether? And show 0.42 directly?

cc @clevinson

Reply:
Or actually, for 0.41, it would be ideal to show a banner on top of the page saying "if you're not using the hub, please upgrade to 0.42". wdyt?

Reply:
Yes i like this approach. Let's only show docs for v0.41 and v0.42, and have v0.41 clearly point out that there is an evidence bug on v0.41.x that renders chains unstable on v0.41.x if they have a non-default bech32 prefix.

Reply:
@lovincyrus can we update this accordingly & merge?  This should also include then an update to the `versions` file, as we now have a `release/v0.41.x` branch, and most stargate docs updates will only be be backported to there.

Reply:
> only show docs for v0.41 and v0.42

Are we only showing v0.41 and v0.42 and get rid of v0.39 and master from the version dropdown?